### PR TITLE
HRINT-2450 [Demo] Update the Demo readme.md file

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,47 @@
...
-
 ```
 npm install
-webpack
+npm run webpack
 ```
 
-# Running the application
-Run the `DemoServer` project in Visual Studio, Rider or `dotnet run`. The application should run on http://localhost:9090/
+Node version needed: v14.18.1
```
Comment (ml054) [inline README.md:8]: 14+ maybe?

File: README.md [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
 ```
 npm install
-webpack
+npm run webpack
 ```
 
-# Running the application
-Run the `DemoServer` project in Visual Studio, Rider or `dotnet run`. The application should run on http://localhost:9090/
+Node version needed: v14.18.1  
+ASP.NET core: 3.1.0
```
Comment (ml054) [inline README.md:9]: 3.1.X ?

File: README.md (hunk 1 above)
Comment (Danielle9897) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -210,3 +210,13 @@ Checklist:
     - ensure it is added as a content file to the `DemoServer` project
     - for non-C# demos: make sure that the `Slug` value matches the corresponding C# demo slug
     - ensure that `SourceFileName` matches the source code file name located in the same folder as `metadata.json`. Please remember to include the file extension.
+
+## When adding a new C# demo:
+
+1. Add demo to `demos.json`
+2. Add slug to `slugs.ts`
+3. Add the demo `controller.cs` file
```
Comment (topalach) [inline README.md:218]: Maybe `controller.cs` should start with upper case? (`Controller.cs`)

Comment (Danielle9897) [reply README.md]: done

**Review by topalach — APPROVED** [review]

**Pull request merged**